### PR TITLE
Python 3 semanage is named python3-libsemanage

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by bastelfreak** [comment]
Hi @ehelms, thanks for the PR. On which operating systems did you test this? I guess this worked on an earlier version of CentOS/RHEL?

**Comment by ehelms** [comment]
I tested this on RHEL 8.

```
Installing:
 policycoreutils-python-utils                           noarch                           2.8-16.1.el8                                                rhel-8-for-x86_64-baseos-rpms                           228 k
Installing dependencies:
 python3-libsemanage                                    x86_64                           2.8-5.el8                                                   rhel-8-for-x86_64-baseos-rpms                           127 k
 python3-policycoreutils                                noarch                           2.8-16.1.el8                                                rhel-8-for-x86_64-baseos-rpms                           2.2 M
 checkpolicy                                            x86_64                           2.8-2.el8                                                   rhel-8-for-x86_64-baseos-rpms                           338 k
 python3-setools                                        x86_64                           4.2.0-2.el8                                                 rhel-8-for-x86_64-baseos-rpms                           598 k
 python3-audit                                          x86_64                           3.0-0.10.20180831git0047a6c.el8                             rhel-8-for-x86_64-baseos-rpms                            85 k
```

**Comment by bastelfreak** [comment]
On CentOS 6 and 7, the old name is valid:
https://rpmfind.net/linux/rpm2html/search.php?query=libsemanage-python

Please keep supporting them and add tests for it.

File: lib/facter/selinux_semanage_is_python3.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Facter.add(:selinux_semanage_is_python3) do
   confine osfamily: 'RedHat'
   setcode do
-    Facter::Core::Execution.execute('rpm -q libsemanage-python3') !~ %r{not installed}
+    Facter::Core::Execution.execute('rpm -q python3-libsemanage') !~ %r{not installed}
```
Comment (ehelms) [inline lib/facter/selinux_semanage_is_python3.rb]: @bastelfreak Note this is *specific* to python3 version of the package, not the python 2 version of the package. CentOS 6 and 7 should play no role in using this fact.

**Comment by ehelms** [comment]
Updated metadata.json

**Comment by ehelms** [comment]
Suppose so if this is dropping support for F26 and F27.

**Comment by bastelfreak** [comment]
According to our review guidelines it's a breaking change.

**Pull request merged**